default.nix

```diff
@@ -137,6 +137,7 @@ rec {
       libusb = null;
       libusb1 = null;
       tinysparql = null;
+      pandoc = null;
```

Comment:
Out of curiosity (I don't speak nix), it pandoc pulled “by default” if you don't set this? Why is it pulled?

Reply:
Oh, sorry, this should get a better commit message.

The issue is that pandoc is (quite reasonably) used by many many projects to compile the docs / man pages / whatever.  It is typically an optional dependency.

We we overlay custom build instructions onto pandocs dependencies then pandoc needs rebuilt.  But pandoc is written in haskell and the rebuild is absolutely massive.  This is one of the reasons our build time for this project exploded.  Some dependency drift caused us to start modifying one of the dependencies of ghc (the haskell compiler) or pandoc and now we need to rebuild the world.  If I null pandoc out of the overlay then no project is allowed to depend on pandoc.  This will skip the docs build (which is fine) and bring our build time back under control.

Reply:
Basically, setting something to null in the overlay makes other builds skip it if it was optional and fail if it was not.  Pandoc is optional for our our relevant packages.